doc/guidelines/frontend-ui-usability.md

```diff
@@ -0,0 +1,177 @@
+# Frontend UI & Usability
+
+## Terminology
+
+### Usability
+
+> Extent to which a system, product or service can be used by specified users to achieve specified goals with effectiveness, efficiency and satisfaction in a specified context of use.
+
+### User Interface (UI)
+
+> Set of all the components of an interactive system that provide information and controls for the user to accomplish specific tasks with the interactive system.
+
+### User Experience (UX)
+
+> Combination of user's perceptions and responses that result from the use and/or anticipated use of a system, product or service.
+
+## Guidelines
+
+This guide should help improve the UI (User Interface) and usability of the PermaplanT web application going forward.  
+The following guidelines are:
+
+- on the one hand a mix of general UI and usability best practices and
+- on the other hand design suggestions tailored to PermaplanT.
+
+### Text
+
+- **Alignment**
+
+  - Left align when text spans over more than three lines because centered text becomes hard to follow when having to jump to the next line multiple times.
```

Comment:
Formulate everything as rule if possible. So is here the rule that we should be below three lines if possible? Which alignment to choose if we are above 3 lines?